### PR TITLE
(iTunesFusion) Moving to core pacakges, along with an update script

### PR DESCRIPTION
**Pull request by The-Running-Dev**

### CODE REVIEW

File: automatic/itunesfusion/itunesfusion.nuspec [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <title>iTunesFusion</title>
+    <authors>Binary Fortress Software</authors>
+    <projectUrl>https://www.itunesfusion.com</projectUrl>
+    <licenseUrl>https://www.itunesfusion.com/License/</licenseUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/eac42b924ab69856d600a4733ea1bba32fbb99d8/icons/iTunesFusion.png</iconUrl>
+    <docsUrl>https://www.itunesfusion.com/Help/</docsUrl>
+    <mailingListUrl>https://www.itunesfusion.com/Discussions/</mailingListUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ## Features
```
Comment (AdmiringWorm) [inline automatic/itunesfusion/itunesfusion.nuspec]: should contain a few lists of description before this Feautres header.

File: automatic/itunesfusion/itunesfusion.nuspec [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    - Configure as many different devices as you need
+    - Synchronize each one with different iTunes playlists.
+    - Use standard playlists or dynamically updating smart playlists to synchronize your device</description>
+    <copyright>Copyright © 2017 Binary Fortress Software</copyright>
+    <tags>itunesfusion itunes sync android mp3 usb</tags>
+    <owners>chocolatey,the-running-dev</owners>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/itunesfusion</packageSourceUrl>
+    <dependencies>
+      <dependency id="chocolatey-core.extension" version="1.2.0" />
+      <dependency id="iTunes" />
```
Comment (AdmiringWorm) [inline automatic/itunesfusion/itunesfusion.nuspec]: add a minimum version for this dependency

File: automatic/itunesfusion/tools/chocolateyInstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿$ErrorActionPreference  = 'Stop'
+
+$downloadDir        = Get-PackageCacheLocation
+$installer          = 'iTunesFusionSetup-2.6.exe'
+$arguments          = @{
+    packageName     = $env:ChocolateyPackageName
+    softwareName    = 'iTunesFusion'
+    file            = Join-Path $downloadDir $installer
```
Comment (AdmiringWorm) [inline automatic/itunesfusion/tools/chocolateyInstall.ps1]: only use the `file` parameter when creating embedded packages.
Remove this please

File: automatic/itunesfusion/update.ps1 [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    Get-RemoteFiles -Purge
+
+    $file = Get-Item tools\*.exe | Select-Object -first 1
+    Remove-Item $file -Force -ErrorAction SilentlyContinue
+}
+
+function global:au_SearchReplace {
+    return @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(?i)(^[$]installer\s*=\s*)('.*')" = "`$1'$([System.IO.Path]::GetFileName($Latest.Url32))'"
```
Comment (AdmiringWorm) [inline automatic/itunesfusion/update.ps1]: Remove this from the replacement (and in the install script).
It isn't needed

File: automatic/itunesfusion/update.ps1 [hunk 5]
```diff
@@ -0,0 +1,37 @@
+﻿Import-Module AU
+
+function global:au_BeforeUpdate {
+    $Latest.ChecksumType32 = 'sha256'
+
+    Get-RemoteFiles -Purge
```
Comment (AdmiringWorm) [inline automatic/itunesfusion/update.ps1]: Don't use Get-RemoteFiles when you don't need to.
Rely on either AU's automatic checksum, or on the function `Get-RemoteChecksum`

File: automatic/itunesfusion/update.ps1 [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    $versionInfo = $downloadUrl -match $versionRegEx
+
+    if ($matches) {
+        $version = $matches[1]
+    }
+
+    return @{ Url32 = $downloadUrl; Version = $version }
+}
+
+Update -ChecksumFor None
```
Comment (AdmiringWorm) [inline automatic/itunesfusion/update.ps1]: change this to `update -ChecksumFor 32`

File: automatic/itunesfusion/itunesfusion.nuspec [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ## Features
+    - Synchronize any iTunes playlist(s) with almost any device
+    - Configure as many different devices as you need
+    - Synchronize each one with different iTunes playlists.
+    - Use standard playlists or dynamically updating smart playlists to synchronize your device</description>
+    <copyright>Copyright © 2017 Binary Fortress Software</copyright>
+    <tags>itunesfusion itunes sync android mp3 usb</tags>
+    <owners>chocolatey,the-running-dev</owners>
```
Comment (AdmiringWorm) [inline automatic/itunesfusion/itunesfusion.nuspec]: who is the original maintainer?
Add him/her also to the owners element.

**Comment by AdmiringWorm** [comment]
also change the filename of the icon to be all lowercase.

**Comment by The-Running-Dev** [comment]
How about now?

**Comment by AdmiringWorm** [comment]
> How about now?

Almost, but there must have been a misunderstanding.
I didn't mean that you should remove the `au_SearchReplace` function, only the line I commented on.

**Pull request merged**